src/ADAL.PCL/Flows/NonInteractive/WsTrustRequest.cs

```diff
@@ -140,7 +140,7 @@ private static void SetKerberosOption(IHttpClient request)
             StringBuilder messageBuilder = new StringBuilder(MaxExpectedMessageSize);
             String schemaLocation = "http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd";
             String soapAction = "http://docs.oasis-open.org/ws-sx/ws-trust/200512/RST/Issue";
-            String rstTrustNamespace = "http://docs.oasis-open.org/ws-sx/ws-trust/200512";
+            String rstTrustNamespace = "http://docs.oasis-open.org/ws-sx/ws-trust/200512/";
```

Comment:
The _ws-trust_ namespace has a trailing slash. See http://docs.oasis-open.org/ws-sx/ws-trust/200512/ws-trust-1.3.xsd